programs/native/rewards/src/lib.rs

```diff
@@ -131,15 +131,15 @@ mod tests {
         )
         .unwrap();
 
-        for _ in 0..vote_program::MAX_VOTE_HISTORY {
-            let vote = Vote::new(1);
+        for i in 0..vote_program::MAX_VOTE_HISTORY - 1 {
```

Comment:
You sure about that? Doesn't look right.

Reply:
yeah, in order for line 139 to be true, you can only submit MAX_VOTE_HISTORY - 1 votes, the next vote will then trigger a reward

Reply:
I thought the Vec was supposed to have a length of MAX_VOTE_HISTORY and that it'd be MAX_VOTE_HISTORY + 1 that gets rewarded

Reply:
I can change it to do that, but I assumed that once a vote got 2^32 lockout, we'd want to pop it for a reward. If we wait for MAX_VOTE_HISTORY + 1, then the vote will have 2^33 lockout

Reply:
MAX_VOTE_HISTORY of 31 is fine. I just don't want these off-by-one errors to creep in because the constant name doesn't reflect what it is.  By HISTORY, we mean length.  Also, VOTE is now the wrong term, right? Maybe MAX_LOCKOUT_HEIGHT?

Reply:
hmmm MAX_LOCKOUT_HEIGHT implies some sort of cap on the maximum slot a vote can be locked out until. I like "MAX_LOCKOUT_HISTORY" or 'MAX_LOCKOUT_DEPTH" b/c it implies the depth of the actual vector holding the lockouts.

Reply:
Can't use "depth" and "locktower". Our towers go up.  LOCKTOWER_HEIGHT? If meh, HISTORY sounds good.